tests/test_repository_tool.py

```diff
@@ -1676,6 +1711,29 @@ def test_revoke(self):
 
 
 
+  def test_check_path(self):
+    # Test that correct path does not raise exception: using '/' as a separator
+    # and does not start with a directory separator
+    self.targets_object._check_path('file1.txt')
+
+    # Test that non-existent path does not raise exception (_check_path
+    # checks only the path string for compliance)
+    self.targets_object._check_path('non-existent.txt')
+    self.targets_object._check_path('subdir/non-existent')
+
+    # Test improperly formatted pathname argument.
+    self.assertRaises(securesystemslib.exceptions.FormatError,
+        self.targets_object._check_path, 3)
+
+    # Test invalid pathname - starting with os separator
+    pathname = os.sep + 'file1.txt'
+    self.assertRaises(tuf.exceptions.InvalidNameError,
+        self.targets_object._check_path, pathname)
```

Comment:
This test will raise for different reasons on *nix and windows, as a consequence we don't trigger the *starts with "/"* condition when testing on windows. (also see comment in `_check_path`)

Reply:
Agree, the commit where I added os.sep in the tests is reverted and instead `_check_path` uses `if pathname.startswith('/')` as suggested, thanks!
Also added an additional test case for paths starting with '\\', just in case for the future.
The new commit: https://github.com/theupdateframework/tuf/pull/1008/commits/e85a3b3d694336d6361ded7aaccd3f45e10fef0f